lib/matplotlib/mlab.py

```diff
@@ -840,6 +846,7 @@ def cohere(x, y, NFFT=256, Fs=2, detrend=detrend_none, window=window_hanning,
     return Cxy, f
 
 
+@_api.deprecated("3.6", alternative="scipy.stats.gaussian_kde")
```

Comment:
We use this internally for violin plots iirc...

Reply:
yeah, I was just abusing CI and waiting for things to fail before figuring what needed to be changed elsewhere.  Worst-case if I can't make KDEs work directly from scipy I'd still suggest we deprecate this as a public method.

Reply:
oh, hold it, do we still not require scipy as a hard requirement?  I guess that means violinplots need us to vendor this?

Reply:
OK backed out this change.  I didn't make this private either because the docs leave it possible to pass a GaussianKDE object to violinplots.